### PR TITLE
[CP-1157] `getFiles` fully handled

### PR DESCRIPTION
**Pull request by dkarski**
Jira: [CP-1157] 

**Description**

This is the core of the USB implementation. The integration isn't fully stable of a few errors attached to the below description as a task to do.  This PR skips tests for parser and decoder logic.

- [mc-usb-device & mc-serial-port-device catalogs decoupled](https://github.com/mudita/mudita-center/pull/697/commits/e7a59bf9a4c386275d6a318d94c35955433b06ca)
- [mc-usb-dependencies resolved](https://github.com/mudita/mudita-center/pull/697/commits/3febaf19d1d5432e4bbb012dfc80194b425a5e01)
- implement `openSession` method for `McUsbDeviceService`
- implement `closeSession ` method for `McUsbDeviceService` 
- implement `getObjectHandles ` method for `McUsbDeviceService` 
- implement `getFileName ` method for `McUsbDeviceService` 
- implement `getFileSize ` method for `McUsbDeviceService` 

This PR no handle
- [CP-1161 - Add tests to Parser](https://appnroll.atlassian.net/browse/CP-1161)
- [CP-1162 - Add test to Decoder class](https://appnroll.atlassian.net/browse/CP-1162)
- [CP-1247 - Implement logic to unmount the device during the opening session process](https://appnroll.atlassian.net/browse/CP-1247)
- [CP-1248 - Implement timeout logic for pending requests](https://appnroll.atlassian.net/browse/CP-1248)
- [CP-1249 - Implement logic to cancel pending requests](https://appnroll.atlassian.net/browse/CP-1249)
- [CP-1250 - Handle application crash during on usb device discovering](https://appnroll.atlassian.net/browse/CP-1250)

[CP-1157]: https://appnroll.atlassian.net/browse/CP-1157?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQe

### CODE REVIEW

File: packages/pure/src/device/device.test.ts [hunk 1]
```diff
@@ -4,20 +4,22 @@
...
-} from "./device.types"
+} from "../index"
 
 let device: MuditaDevice
 
 beforeEach(async (done) => {
   mockSerialPort()
 
   const devices = await PureNode.getDevices()
+  console.log("devices: ", devices)
```
Comment (igorbezsmertnyi) [inline packages/pure/src/device/device.test.ts:22]: Let's remove it

File: packages/pure/src/mc-usb-device/usb-parser.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      bytes.setUint32(12 + index * 4, element, true)
+    })
+    logger.info(`==== usb: buildContainerPacket: ====`)
+    console.log(buffer)
+    return buffer
+  }
+
+  static parseContainerPacket(buffer: ArrayBuffer): UsbResponse {
+    logger.info(`==== usb: parseContainerPacket: ====`)
+    console.log(buffer)
```
Comment (igorbezsmertnyi) [inline packages/pure/src/mc-usb-device/usb-parser.ts:32]: Please remove it

File: packages/pure/src/device/device.test.ts (hunk 1 above)
Comment (dkarski) [reply packages/pure/src/device/device.test.ts]: https://github.com/mudita/mudita-center/pull/697/commits/32a9172dd4cf1699b3d0d7babaa727fdce4e8e10

File: packages/pure/src/mc-usb-device/usb-parser.ts (hunk 2 above)
Comment (dkarski) [reply packages/pure/src/mc-usb-device/usb-parser.ts]: https://github.com/mudita/mudita-center/pull/697/commits/32a9172dd4cf1699b3d0d7babaa727fdce4e8e10

**Review by wercix — APPROVED** [review]

**Review by igorbezsmertnyi — APPROVED** [review]

**Pull request merged**